Testscripts/Linux/SetupRDMA.sh

```diff
@@ -180,27 +197,35 @@ function Main() {
 			LogMsg "Required 32-bit java"
 			dpkg --add-architecture i386
 
-			install_package "build-essential python-setuptools libibverbs-dev bison flex ibverbs-utils net-tools libdapl2 rdmacm-utils bc"
+			req_pkg="build-essential python-setuptools libibverbs-dev bison flex ibverbs-utils net-tools libdapl2 rdmacm-utils bc"
+			install_package $req_pkg
+			LogMsg "Installed the required packaes, $req_pkg"
 			os_RELEASE=$(awk '/VERSION_ID=/' /etc/os-release | sed 's/VERSION_ID=//' | sed 's/\"//g')
 			if [ $mpi_type == "ibm" ]; then
 				if [[ "$os_RELEASE" == "18.04" ]]; then
-					install_package "openjdk-8-jdk:i386"
+					req_pkg="openjdk-8-jdk:i386"
 				else
-					install_package "openjdk-9-jre:i386"
+					req_pkg="openjdk-9-jre:i386"
 				fi
+				install_package $req_pkg
+				LogMsg "IBM MPI required 32-bit Java in the system, $req_pkg"
 			fi
-			LogMsg "*** Adding kernel modules to /etc/modules"
+			LogMsg "Adding kernel modules to /etc/modules"
 			echo rdma_ucm >> /etc/modules
 			modprobe rdma_ucm
+			LogMsg "Loaded rdma_ucm successfully"
```

Comment:
if $? == 0, then this command executed successfully.

Reply:
fixed in 66ae117079544b049a7a20e2d236a40d7b8e8825